lib/count-users-by-year-for-deletion.rb

```diff
@@ -8,28 +8,27 @@
   config.retry = true
 end
```

Comment:
Can we replace these lines (3-9) with the following code.

```
require_relative 'zendesk-setup.rb'
```

Reply:
Resolved as suggested.